catboost/python-package/ut/medium/test.py

```diff
@@ -11335,6 +11335,22 @@ def evaluate(self, approxes, target, weight):
 
             return error_sum, weight_sum
 
+        def gpu_evaluate(self, approx, target, weight, output, output_weight):
+            print(1 / 0)
```

Comment:
Why?

Reply:
I was just curious, wanted to know, what it would print 😁